src/PureCloud.Client/Apis/NotificationsApi.cs

```diff
@@ -155,4 +155,18 @@ public async Task DeleteNotificationsChannelSubscriptionsAsync(string channelId,
 
         response.EnsureSuccessStatusCode();
     }
+
+    /// <inheritdoc />
+    public async Task<bool> VerifyNotificationsChannelAsync(string channelId, CancellationToken cancellationToken = default)
```

Comment:
Does this method return Boolean in the old file or a concrete type?

Reply:
Return the same type as the old method does. When there is no content, the Total items will just be 0 and the records and empty collection

Reply:
Looking back at the old method, I agree with your original approach. Please return book instead